src/sst/elements/vanadis/inst/vmovci.h

```diff
@@ -0,0 +1,109 @@
+// Copyright 2009-2021 NTESS. Under the terms
+// of Contract DE-NA0003525 with NTESS, the U.S.
+// Government retains certain rights in this software.
+//
+// Copyright (c) 2009-2021, NTESS
+// All rights reserved.
+//
+// Portions are copyright of other developers:
+// See the file CONTRIBUTORS.TXT in the top level directory
+// the distribution for more information.
+//
+// This file is part of the SST software package. For license
+// information, see the LICENSE file in the top level directory of the
+// distribution.
+
+#ifndef _H_VANADIS_MOVI
+#define _H_VANADIS_MOVI
+
+#include "inst/vinst.h"
+
+namespace SST {
+namespace Vanadis {
+
+class VanadisMoveCompareImmInstruction : public VanadisInstruction {
+public:
+    VanadisMoveCompareImmInstruction(const uint64_t addr, const uint32_t hw_thr, const VanadisDecoderOptions* isa_opts,
```

Comment:
idiomatic C++ doesn't make values const (especially not built in values like int types or float).  It shouldn't change function signature or code gen to remove these consts on the values.